zulipterminal/config/tutorial.py

```diff
@@ -0,0 +1,18 @@
+# Tutorial to display at start
+TUTORIAL = [
```

Comment:
To echo @shreyamalviya's point previously, after this text (independently of showing the tutorial text), I think it would be good to show a '"Remember you can use '?' to explore the available shortcut/hot keys" message or similar. That would act as a 'look <here> for more keys than we just introduced' as well as a 'remember you can get help <here>' if the tutorial is not enabled.